_episodes/04-pipefilter.md

```diff
@@ -26,8 +26,8 @@ keypoints:
 Now that we know a few basic commands,
 we can finally look at the shell's most powerful feature:
 the ease with which it lets us combine existing programs in new ways.
-We'll start with a directory called `molecules`
-that contains six files describing some simple organic molecules.
+Within the `data-shell` directory, there is a directory called `molecules`
+which contains six files describing some simple organic molecules.
```

Comment:
Good stuff!

Reply:
Thanks, when I was teaching this, it definitely took me a second to remember where the molecules folder was.